### ISSUE TITLE
can not lint scss ?

### ISSUE THREAD

**Issue by daizhan** [shinnn/vscode-stylelint#19]
stylelint plugin can not lint my scss file. 
all vs code config is default. 
`test.scss`
```scss
p{
    color: #FFF;
}
```
`.stylelintrc`
```json
{
    "rules": {
        "color-hex-case": "lower",
        "block-opening-brace-space-before": "always",
        "declaration-colon-space-after": "always"
    }
}
```
there is no error or warning in vscode.
but it works if I modify the exitension of scss file to css， there are three errros.

stylelint result （env :  win10, stylelint 6.8.0, stylelint-plugin 0.11.3）
`project structure`
 ![7da0 tmp](https://cloud.githubusercontent.com/assets/5318547/16797016/1e094cb6-4918-11e6-9fbf-937b754d47d5.png)
`vscode scss`
![4d47 tmp](https://cloud.githubusercontent.com/assets/5318547/16796995/03ab8e7e-4918-11e6-9429-1f4f27923c49.png)
`vscode css`
![a8f7 tmp](https://cloud.githubusercontent.com/assets/5318547/16796997/0796c56c-4918-11e6-9f02-3ea599092916.png)
`powershell stylelint CLI`
![37c tmp](https://cloud.githubusercontent.com/assets/5318547/16797000/0b6bc570-4918-11e6-80b6-261e2178a828.png)

**Comment by shinnn** [comment]
Fixed in vscode-stylelint v0.12.1. Thanks for the report.
